modules/ROOT/pages/6.8.2-release-notes.adoc

```diff
@@ -0,0 +1,83 @@
+= TinyMCE 6.8.2
+:navtitle: TinyMCE 6.8.2
+:description: Release notes for TinyMCE 6.8.2
+:keywords: releasenotes, new, changes, bugfixes
+:page-toclevels: 1
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+[[overview]]
+== Overview
+
+{productname} 6.8.2 was released for {enterpriseversion} and {cloudname} on Wednesday, December 13^th^, 2023. These release notes provide an overview of the changes for {productname} 6.8.2, including:
+
+* xref:accompanying-premium-plugin-changes[Accompanying Premium plugin changes]
+* xref:bug-fixes[Bug fixes]
+
+
+[[accompanying-premium-plugin-changes]]
+== Accompanying Premium plugin changes
+
+The following premium plugin updates were released alongside {productname} 6.8.2.
+
+=== PowerPaste 6.2.4
+
+The {productname} 6.8.2 release includes an accompanying release of the **PowerPaste** premium plugin.
+
+**PowerPaste** 6.2.4 includes the following fix.
+
+==== `white-space-collapse` CSS property was not correctly supported when pasting from Google Docs.
```

Comment:
@mkzhx should we add links to [mdn-css_properties_white-space-collapse](https://caniuse.com/mdn-css_properties_white-space-collapse) and the below for this?

- [#white-space-property](https://www.w3.org/TR/css-text-3/#white-space-property)
- [#white-space-collapsing](https://www.w3.org/TR/css-text-4/#white-space-collapsing)

Reply:
Yes that's a good idea, I think just the W3 spec links are sufficient.